runtime/exec/compact.go

```diff
@@ -39,6 +39,9 @@ func Compact(ctx context.Context, lk *lake.Root, pool *lake.Pool, branchName str
 	slicer := meta.NewSlicer(lister, zctx)
 	puller := meta.NewSequenceScanner(octx, slicer, pool, nil, nil, nil)
 	w := lake.NewSortedWriter(ctx, zctx, pool)
+	if writeVectors {
+		w.EnableVectorWrite()
+	}
```

Comment:
Nit: Just pass this flag to NewSortedWriter?
```suggestion
	w := lake.NewSortedWriter(ctx, zctx, pool, writeVectors)
```

Reply:
ok